internal/runtime/funcs/image_decode.go

```diff
@@ -0,0 +1,49 @@
+package funcs
+
+import (
+	"context"
+	"image"
+	"image/png"
+	"strings"
+
+	"github.com/nevalang/neva/internal/runtime"
+)
+
+type imageDecode struct{}
```

Comment:
I'm not sure if all of these are used in examples but in case they're not, I suggest only add those runtime funcs that are actually used

Reply:
I need an example for read/write on image

Reply:
Removed for now...